File: TP1/BonRevoirObjC/BonRevoirObjC/ViewController.m
//
//  ViewController.m
//  BonRevoirObjC
//
//  Created by m2sar on 19/09/2015.
//
//  @author Ilyas  Toumlilt - <ilyas.toumlilt@etu.upmc.fr>
//  @author Adrien Ferreira - <adrien.ferreira@etu.upmc.fr>
//
//  Copyright (c) 2015 m2sar. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    _myLabel.text = @"";
    [_myButton setTitle:@"Dis Bonjour !" forState:UIControlStateNormal];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)onClickMyButton:(id)sender {
    if ([_myButton.titleLabel.text isEqualToString:@"Dis Bonjour !"] ){
        [_myLabel setText:@"Bonjour!!!"];
        [_myButton setTitle:@"Dis Au Revoir !" forState:UIControlStateNormal];
    } else {
        [_myLabel setText:@"Au Revoir!!!"];
        [_myButton setTitle:@"Dis Bonjour !" forState:UIControlStateNormal];
    }
}
@end

File: TP1/BonRevoirSwift/BonRevoir/Bonrevoir/ViewController.swift
//
//  ViewController.swift
//  Bonrevoir
//
//  Created by m2sar on 21/09/2015.
//  Copyright (c) 2015 m2sar. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var Label: UILabel!
    
    var boolean : Bool = true
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        Label.sizeToFit()
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func Bouton(sender: AnyObject) {
        boolean = !boolean
        if(boolean){
            Label.text = "Au revoir"
            sender.setTitle("Dis Bonjour", forState: .Normal)
        }
        else{
            Label.text = "Bonjour"
            sender.setTitle("Dis Au Revoir", forState: .Normal)
        }
    }

}

